### ISSUE TITLE
idea/todo direct control

### ISSUE THREAD

**Issue by HyperSoop** [Stellarica/Stellarica-Core#9]
In direct/immersive/action control mode, put the client so sit in a boat sitting on a marker entity; then detect WASD and mosue wheel.
WASD is strafe (camera relative), AD can be double clicked to turn, use F to toggle cruise mode (speed is kept indefinitely unless compensated for). Use the mousewheel to ascend and descend. 

This is a long term thing to do, but an interesting one nontheless.

**Comment by HyperSoop** [comment]
I wonder if it would be possible to turn the ship by turning your camera instead of on double tab.

**Comment by trainb0y** [comment]
yeah that’s probably possible, but it might be a little unintuitive. Either way we do need to improve on the current controls.

**Comment by HyperSoop** [comment]
It could be a toggleable option. Just turning your camera would feel more intuitive than double tapping though, and also reduce the disorientation caused by sharp 90-degree turns (since you'd only rotate the ship and the player's view angle would stay the same)
